DFC.ServiceTaxonomy.Cypher/Configuration/EndpointConfiguration.cs

```diff
@@ -0,0 +1,11 @@
+namespace DFC.ServiceTaxonomy.Cypher.Configuration
```

Comment:
It'd be nice if the project was called something more appropriate (reports?) rather than Cypher.